### PR TITLE
Checkstyle

### PR DESCRIPTION
**Pull request by AustinShalit**
There is still more work to be done but this is a good start. 

Depends on #38

### CODE REVIEW

**Comment by AustinShalit** [comment]
I would like to merge this as is and continue to do other clean up things in other PRs.  This is just the starting point.

File: src/main/java/edu/wpi/first/smartdashboard/gui/DashboardPanel.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 import javax.swing.SwingUtilities;
 
 /**
- * This is the main panel, it sits within the {@link DashboardFrame} and
+ * This is the SmartDashboard panel, it sits within the {@link DashboardFrame} and
```
Comment (PeterJohnson) [inline src/main/java/edu/wpi/first/smartdashboard/gui/DashboardPanel.java]: Did you mean to change this one?

File: src/main/java/edu/wpi/first/smartdashboard/gui/elements/RobotPreferences.java [hunk 2]
```diff
@@ -478,6 +470,7 @@ public Object validateValue(String value, int typeIndex) {
             JOptionPane.showMessageDialog(RobotPreferences.this,
                 "Invalid string: " + e.getMessage(), "Bad Value", JOptionPane.ERROR_MESSAGE);
           }
+        // TODO: Should this be a fallthrough?
```
Comment (PeterJohnson) [inline src/main/java/edu/wpi/first/smartdashboard/gui/elements/RobotPreferences.java]: No, this should not be a fallthrough.

File: src/main/java/edu/wpi/first/smartdashboard/livewindow/elements/LWSubsystem.java [hunk 3]
```diff
@@ -22,7 +22,7 @@
 import javax.swing.SwingUtilities;
 
 /**
- * The main player in the Live Window. Subsystems hold every component that
+ * The SmartDashboard player in the Live Window. Subsystems hold every component that
```
Comment (PeterJohnson) [inline src/main/java/edu/wpi/first/smartdashboard/livewindow/elements/LWSubsystem.java]: Did you mean to change this one?

File: src/main/java/edu/wpi/first/smartdashboard/gui/elements/RobotPreferences.java (hunk 2 above)
Comment (AustinShalit) [inline src/main/java/edu/wpi/first/smartdashboard/gui/elements/RobotPreferences.java]: Ok.  I am going to fix it in a separate PR.

**Comment by AustinShalit** [comment]
Actually, can this supersede #38 instead of depending on it?

**Pull request merged**